### PR TITLE
sample: don't restore passthrough devices on exit by default

### PR DESCRIPTION
**Pull request by totoroow**
Passthroughed devices won't be restored back to SOS once launch
script exits by default. However you could revert this behavior
by specifying '-r' parameter if you want.

Signed-off-by: Tw <wei.tan@intel.com>
Tracked-On: #4514

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by fyin1** [comment]
What about adding the logger option when launch RTVM?

**Comment by wenlingz** [comment]
@ywan170 please help approve this PR, thanks

**Comment by ywan170** [comment]
@binbinwu1 Please help take a look.

**Comment by totoroow** [comment]
@fyin1 Done.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: devicemodel/samples/nuc/launch_xenomai.sh [hunk 1]
```diff
@@ -83,21 +83,22 @@ function de_passthru_pci_devs()
...
-trap de_passthru_pci_devs EXIT
-
 mem_size=1024M
 eth=
 disk=
 img=
 kernel=
 passthrus=()
-while getopts hn:b:f:k:B: opt
+while getopts hn:b:f:k:B:r opt
```
Comment (binbinwu1) [inline devicemodel/samples/nuc/launch_xenomai.sh:92]: @totoroow Could you also update the usage to show the option 'r'?

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**